include/revng/Model/Configuration.h

```diff
@@ -17,6 +17,9 @@ type: struct
   - name: Naming
     type: NamingConfiguration
     optional: true
+  - name: Miscellaneous
+    type: MiscellaneousConfiguration
+    optional: true
```

Comment:
I'd flatten `Miscellaneous` in here, we can pack things into a sub-struct in the future if there's a couple of options that can be grouped together.

Reply:
Done!